app/models/rental.py

```diff
@@ -1,4 +1,12 @@
+from sqlalchemy.orm import backref
```

Comment:
We don't need this import. For simple backrefs, the relationship named parameter is sufficient. If we _do_ need the backref symbol (used to set more complex options on a backref), we can get at it though the `db` reference. Sometimes we do need to import stuff directly from SQLAlchemy, but as much as possible, we should try to bring in SQL-related symbols through the `db` ref we already have.